api/stages_api.py

```diff
@@ -105,12 +105,12 @@ def _update_stage_vals(self, stage: Stage, feature_type: int,
       if field in body:
         if stage.milestones is None:
           stage.milestones = MilestoneSet()
-        setattr(stage.milestones, field, body['field'])
+        setattr(stage.milestones, field, body[field])
```

Comment:
This seems like a pretty significant bug fix.   Was the old code ever getting run?

Reply:
Yeah, this is a large bug. 😅 
Luckily, this code is only being run at the moment when creating a new blank stage. PATCH requests are not wired up yet, which is why we haven't seen any issue yet. Thanks for catching this @KyleJu!

Reply:
Caught it along the way when I wrote the tests. Anytime!